src/AudioInterface.cpp

```diff
@@ -41,21 +41,25 @@
 #include <cmath>
 #include <iostream>
 
+#include "InputMixMode.h"
 #include "JackTrip.h"
 
 using std::cout;
 using std::endl;
 
 //*******************************************************************************
-AudioInterface::AudioInterface(JackTrip* jacktrip, int NumInChans, int NumOutChans,
+AudioInterface::AudioInterface(JackTrip* jacktrip, int BaseInChan, int NumInChans,
+                               int NumOutChans, int InputMixMode,
```

Comment:
```suggestion
                               int NumOutChans, InputMixMode InputMixMode,
```

If you use InputMixMode everywhere, you don't need to static_cast anymore.